HIRS_ProvisionerTPM2/test/Utils_Test.cpp

```diff
@@ -536,6 +536,26 @@ TEST_F(UtilsTest, ParseTpm2ToolsVersionFailNonSemanticVersion) {
     ASSERT_EQ(expectedOutput, version);
 }
 
+TEST_F(UtilsTest, ParseTpm2ToolsMajorVersionSuccess) {
+    stringstream versionOutput;
+    versionOutput << "3.0.1";
+
+    string expectedOutput = "3";
+    string majorVersion = Tpm2ToolsOutputParser::parseTpm2ToolsMajorVersion(
+            versionOutput.str());
+    ASSERT_EQ(expectedOutput, majorVersion);
+}
+
+TEST_F(UtilsTest, ParseTpm2ToolsMajorVersionSuccessMultiNumeralVersion) {
+    stringstream versionOutput;
+    versionOutput << "10.29.970";
+
+    string expectedOutput = "10";
+    string majorVersion = Tpm2ToolsOutputParser::parseTpm2ToolsMajorVersion(
+            versionOutput.str());
+    ASSERT_EQ(expectedOutput, majorVersion);
+}
+
```

Comment:
Could you add a test here that ensures `parseTpm2ToolsMajorVersion` returns an empty string if it can't parse its argument?